### PR TITLE
created self-host page #858

### PR DESCRIPTION
**Pull request by akshay-bharadva**
### issue : SigNoz/signoz-web#123 

- created self-host signup page

### CODE REVIEW

**Comment by ankit01-oss** [comment]
Hi @akshay-bharadva , 
Thanks for the PR. Can you also include some basic validation checks in the form?

File: src/css/custom.css [hunk 1]
```diff
@@ -389,3 +389,140 @@ table.custom-table {
 table.custom-table tbody tr {
   transition: box-shadow 0.2s;
 }
+/* email subscribe */
+.email-subscribe{
```
Comment (palashgdev) [inline src/css/custom.css:393]: can you please make a separate CSS file in which you can import this one

File: src/css/custom.css [hunk 2]
```diff
@@ -389,3 +389,140 @@ table.custom-table {
...
+  color: #000000;
```
Comment (palashgdev) [inline src/css/custom.css:394]: can we make sure we use design tokens which are already configured

File: src/css/custom.css [hunk 3]
```diff
@@ -389,3 +389,140 @@ table.custom-table {
...
+  font-family: 'inline', sans-serif;
+  font-size: 16px;
```
Comment (palashgdev) [inline src/css/custom.css:396]: use rem everywhere

File: src/pages/self-host.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+              </div>
+            </div>
+            <div className="col col--6">
+              <div className="card-demo margin--md">
+                <div className="card">
+                  <div className="card__body">
+                    <p className="text-center form-title">
+                      Get more info on SigNoz Enterprise
+                    </p>
+                    <input
```
Comment (palashgdev) [inline src/pages/self-host.js:61]: can we have common input as we are repeating outself

File: src/pages/self-host.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                      type="text"
+                      className="text-input"
+                      name="email"
+                      placeholder={"Email*"}
+                      {...register("email", {
```
Comment (palashgdev) [inline src/pages/self-host.js:66]: let's declare constant for same

File: package.json [hunk 6]
```diff
@@ -24,6 +24,7 @@
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "react-ga": "^3.3.0",
+    "react-hook-form": "^7.43.8",
```
Comment (palashgdev) [inline package.json:27]:
```suggestion
    "react-hook-form": "7.43.8",
```

**Review by palashgdev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**